charts/node-analyzer/README.md

```diff
@@ -209,6 +209,9 @@ The following table lists the configurable parameters of the Sysdig Node Analyze
 | `nodeAnalyzer.hostScanner.resources.limits.ephemeral-storage`        | Specifies the Host Scanner Storage limit per node.                                                                                                                                          | `1Gi`                                                                                                                                                                 |
 | `nodeAnalyzer.hostScanner.sslVerifyCertificate`                                  | Set to `false` to allow insecure connections to the Sysdig backend, such as an On-Prem deployment.                                                                                          |                                                                                                                                                                       |
 | `nodeAnalyzer.hostScanner.probesPort`                                | Specifies the port where readiness and liveness probes are exposed.                                                                                                                         | `7001`                                                                                                                                                                |
+| `nodeAnalyzer.hostScanner.scanContainers.enabled`                           | Set to `true` to scan containers                                                                                                                                                    | `false`                                                                                                                                                             |
+| `nodeAnalyzer.hostScanner.scanContainers.dockerSocketPath`          | Specifies the path to docker socket                                                                                                                                                 | `unix:///var/run/docker.sock`                                                                                                                                                                  |
+| `nodeAnalyzer.hostScanner.scanContainers.podmanSocketPath`          | Specifies the path to podman socket                                                                                                                                                 | `unix:///var/run/podman.sock`                                                                                                                                                                  |
```

Comment:
I see this is commented in the values file. Is the default still valid?

Reply:
Yes, see my comment above.